MergeCalendarsTogether.gs

```diff
@@ -246,11 +289,7 @@ function RetrieveCalendars(startTime, endTime) {
       items.push(...result.items)
       nextPage = result.nextPageToken;
     } while(nextPage);
-    console.log(`Found ${items.length} items for ${calendarId}`)
-    const isNoEventsFound = !items.length
-    if (isNoEventsFound) {
-      return;
-    }
```

Comment:
@Flare576 Maybe you can take a look at this and verify what the goal was in case I'm breaking a different edge case.  The end result I experienced is that if one of the calendars is empty, it won't return as a valid calendar at all. 

Since the calendars returned in the array are also used as destinations, the empty calendar would  then receive no events even if it was missing events from another calendar.

Reply:
Only situation I can think of is if this was a catch for a calendar without access, but I believe it process will actually fail out earlier now than this point. Removing this check seems logical!